src/sentry/static/sentry/app/views/alerts/rules/index.tsx

```diff
@@ -115,14 +114,17 @@ class AlertRulesList extends AsyncComponent<Props, State & AsyncComponent['state
     return (
       <Layout.Body>
         <Layout.Main fullWidth>
-          <Panel>
-            <PanelHeader>
-              <TableLayout>
-                <div>{t('Type')}</div>
-                <div>{t('Alert Name')}</div>
-                <div>{t('Project')}</div>
-                <div>{t('Created By')}</div>
-                <div>
+          {loading ? (
+            <LoadingIndicator />
+          ) : (
```

Comment:
You can give `<PanelTable>` `isLoading` prop which will handle the loading state for you

Reply:
updated